src/Restaurant.js

```diff
@@ -1,13 +1,29 @@
-import React from "react";
-
+import React, { useState } from "react";
+import Order from "./Order";
+import "./Restaurant";
 const Restaurant = () => {
-  const pizzas = 0;
   return (
     <div>
       <h3>Restaurant Orders</h3>
-      <ul>
+      <ul
+        style={{
+          display: "flex",
+          flexWrap: "nowrap",
+          justifyContent: "space-between",
+          listStyle: "none"
+        }}
+      >
+        <li>
+          {" "}
+          <Order orderType="pizza" />{" "}
+        </li>
+
+        <li>
+          <Order orderType="salad" />
+        </li>
         <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
+          {" "}
+          <Order orderType="cake" />
```

Comment:
👍  - Looks good here, 3 separate components for the different Order types! Is there a way we could create all 3 components without having to declare the same component 3 times?